docs/usage/model_config.md

```diff
@@ -346,3 +350,40 @@ print(Model(x=[1, '2']))
 print(Model(x=[1, 2]))
 #> x=[1, 2]
 ```
+
+
+## Protected Namespaces
+
+Protected namespaces prevent model from having field that starts with them.
+It can be a list of strings and the default value is `('model_', )`.
```

Comment:
```suggestion
Pydantic prevents collisions between model attributes and `BaseModel`'s own methods by namespacing them with the prefix `model_` and preventing setting fields or extra fields that start with that prefix.
This is customizable using the `protected_namespaces` option in the model's config so that you can allow overriding `model_` or add your own protected namespaces.
```